src/main/java/net/mcreator/ars_technica/common/glyphs/EffectWhirl.java

```diff
@@ -83,7 +75,7 @@ protected void resolve(Vec3 position, Level world, @NotNull LivingEntity shooter
         double durationAmplifier = spellStats.getDurationMultiplier();
         int extraDurationTicks = Math.toIntExact(Math.round(durationAmplifier * 40));
 
-        FanProcessingType processingType = AllFanProcessingTypes.NONE;
+        FanProcessingType processingType = new AllFanProcessingTypes.SplashingType();
```

Comment:
```suggestion
        FanProcessingType processingType = null
```
Maybe we need the `@Nullable` annotation